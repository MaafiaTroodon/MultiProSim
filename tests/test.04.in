2 5 2
Proc1 3 1 1
SEND 201
RECV 201
HALT

Proc2 3 1 2
RECV 101
SEND 101
HALT
