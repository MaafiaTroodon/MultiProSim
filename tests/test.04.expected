[01] 00000: process 1 new
[01] 00000: process 1 ready
[01] 00000: process 1 running
[01] 00001: process 1 blocked (send)
[01] 00002: process 1 ready
[01] 00002: process 1 running
[01] 00003: process 1 blocked (recv)
[01] 00004: process 1 finished
[02] 00000: process 1 new
[02] 00000: process 1 ready
[02] 00000: process 1 running
[02] 00001: process 1 blocked (recv)
[02] 00002: process 1 ready
[02] 00002: process 1 running
[02] 00003: process 1 blocked (send)
[02] 00004: process 1 finished
| 00004 | Proc 01.01 | Run 2, Block 0, Wait 0, Sends 1, Recvs 1
| 00004 | Proc 02.01 | Run 2, Block 0, Wait 0, Sends 1, Recvs 1
